backend/src/db/migrations/20200312140328-bulk_upload_to_s3.js

```diff
@@ -32,39 +32,38 @@ export async function up(next) {
   try {
     // Implement your migration here.
     const { records } = await transaction.run('MATCH (image:Image) RETURN image.url as url')
-    let urls = records.map(r => r.get('url'))
-    urls = urls.filter(url => url.startsWith('/uploads'))
+    let urls = records.map((r) => r.get('url'))
+    urls = urls.filter((url) => url.startsWith('/uploads'))
     const locations = await Promise.all(
       urls
-        .map(url => {
+        .map((url) => {
           return async () => {
             const { pathname } = new URL(url, 'http://example.org')
             const fileLocation = path.join(__dirname, `../../../public/${pathname}`)
             const s3Location = `original${pathname}`
-            const mimeType = mime.lookup(fileLocation)
+            if (existsSync(fileLocation)) {
```

Comment:
Can we maybe do sth. in `else`? E.g. delete the `url` from the Image node? Shall we do that in another migration?

Reply:
I thought about that, like we shouldn't have any `Image` nodes that point to files that don't exist in the `fileSystem`... like maybe we could have done that in the migration that created the `Image` nodes, but since that has already been deployed, we could iterate through all `Image`, check if they exist  in the file system and if not delete them and their relationship to `User/Post`, oder?